### PR TITLE
fix: make root sub org unit selection work

### PR DESCRIPTION
**Pull request by Mohammer5**
Closes LIBS-254 ([link](https://jira.dhis2.org/browse/LIBS-254))

- [x] Works with multiple roots on different hierarchy levels
- [x] Tested with cypress (for details see additions/changes in the feature files)

### CODE REVIEW

File: components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js [hunk 1]
```diff
@@ -177,6 +191,7 @@ OrganisationUnitNode.propTypes = {
     dataTest: PropTypes.string.isRequired,
     id: PropTypes.string.isRequired,
     renderNodeLabel: PropTypes.func.isRequired,
+    rootId: PropTypes.string.isRequired,
```
Comment (varl) [inline components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js:194]: This addition should be a `feat` if it's part of the public API for the component.

**Review by varl — APPROVED** [review]
Nice job fixing this without breakages. I noticed that there is an addition to the API of the OrgUnitNode component, which seems like a feature ?

File: components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js (hunk 1 above)
Comment (Mohammer5) [reply components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js:194]: This is internal. The component we expose is the `OrganisationUnitTree`, which renders the `OrganisationUnitNode` and provides the `rootId` [here](https://github.com/dhis2/ui/blob/c05de2bbd90a3293af698917ec44d93e3423fd22/components/organisation-unit-tree/src/organisation-unit-tree/organisation-unit-tree.js#L84).

Comment (varl) [reply components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js:194]: Ah ! Great, thanks for clarifying.

**Pull request merged**